src/App.tsx

```diff
@@ -6,19 +6,23 @@ import { store, persistor } from '@/Store'
 import ApplicationNavigator from '@/Navigators/Application'
 import './Translations'
 
-const App = () => (
-  <Provider store={store}>
-    {/**
-     * PersistGate delays the rendering of the app's UI until the persisted state has been retrieved
-     * and saved to redux.
-     * The `loading` prop can be `null` or any react instance to show during loading (e.g. a splash screen),
-     * for example `loading={<SplashScreen />}`.
-     * @see https://github.com/rt2zz/redux-persist/blob/master/docs/PersistGate.md
-     */}
-    <PersistGate loading={null} persistor={persistor}>
-      <ApplicationNavigator />
-    </PersistGate>
-  </Provider>
-)
+const App = () => {
+  const unusedVariable = 'unusedVariable'
```

Comment:
<table data-meta="generated_by_danger-pr">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vkundapur/ci-sample/blob/2d4416415ae88720c3231924e1aff2fb37820759/src/App.tsx#L10"></span>'unusedVariable' is assigned a value but never used.</td>
    </tr>
  </tbody>
</table>